ush/machine/hera.yaml

```diff
@@ -35,7 +43,21 @@ platform:
   FIXsfc: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_sfc_climo
   FIXshp: /scratch2/BMC/det/UFS_SRW_App/develop/NaturalEarth
   EXTRN_MDL_DATA_STORES: hpss aws nomads
-
+  ARCHIVEDIR: ""
+  NCARG_ROOT: ""
+  NCL_HOME: ""
+  OBSPATH: "/scratch2/BMC/zrtrr/rli/data/obs_rap"
+  OBSPATH_NSSLMOSIAC: "/scratch2/BMC/zrtrr/rli/data/reflectivity"
+  LIGHTNING_ROOT: "/scratch2/BMC/zrtrr/rli/data/lightning"
+  ENKF_FCST: "/scratch2/BMC/zrtrr/rli/data/enkf/atm"
+  FFG_DIR: ""
+  SST_ROOT: "/scratch2/BMC/zrtrr/rli/data/highres_sst"
+  GVF_ROOT: "/scratch2/BMC/zrtrr/rli/data/gvf/grib2"
+  IMSSNOW_ROOT: "/scratch2/BMC/zrtrr/rli/data/snow/ims96/grib2"
+  RAPHRR_SOIL_ROOT: ""
+  FIRE_RAVE_DIR: ""
+  AIRCRAFT_REJECT: "/scratch2/BMC/zrtrr/rli/data/amdar_reject_lists"
+  SFCOBS_USELIST: "/scratch2/BMC/zrtrr/rli/data/mesonet_uselists"
```

Comment:
I figured this from the `set_rrfs_config.sh` file in GSL fork for the test case I was running. Depending on the type of test these can be different.

Reply:
I was curious how you were going to handle switching between a retro and non-retro run. As you pointed out, `set_rrfs_config.sh` can switch these paths. My thoughts where to either add jinja logic to switch paths or add a new yaml section like "platform" or "data" called "retro_run", which will reconfigure paths whenever a retro task will run.

Reply:
@EdwardSnyder-NOAA I haven't thought that far ahead , just picked what I needed for the CONUS test case!
Providing multiple paths in the machine yaml files for retro/non-retro realtime/retro and other conditions, similar to how the `data:` section picks a path based on the format, could probably work. Now that jinja is supported in the machine file too, that is also a possibility, but I prefer the other option. I will take a look at the latest `set_rrfs_config/genera_config` files to see how best to handle it.

Reply:
I have implemented the yaml solution by extending the `data:` section to contain both realtime and retrospective run paths. Each RRFS data path is still under `platform:` and will be set in `setup.py` during workflow generation. One can still override the values by setting them directly. Let me know what you think.

Reply:
@danielabdi-noaa - that looks good to me!